ssh.py

```diff
@@ -36,32 +36,27 @@ def show_wofi(hosts):
     command="wofi -p \"SSH hosts: \" -d -i --hide-scroll"
     
     process = subprocess.Popen(command,shell=True,stdin=subprocess.PIPE,stdout=subprocess.PIPE)
-    return process.communicate(input=hosts)[0]
-
-# Returns the id of the window that was selected by the user inside wofi
-def parse_id(hosts, selected):
-    selected = int(selected.decode("UTF-8"))
-    return str(hosts[selected].get('id'))
+    ret = process.communicate(input=hosts)
+    host, rest = ret
+    return host
 
 # Switches the focus to the given id
-def ssh_to_host(host, terminal):
-    command=terminal + " 'ssh " + host + "'"
-    
+def ssh_to_host(host, terminal, ssh_command):
+    command = "{terminal} \'{ssh_command} {host}\'".format(terminal=terminal, ssh_command=ssh_command, host=host)
     process = subprocess.Popen(command,shell=True)
 
 # Entry point
 if __name__ == "__main__":
     
     parser = ArgumentParser(description="Wofi based ssh launcher")
-    parser.add_argument("terminal")
+    parser.add_argument("terminal", help='Terminal command to use')
+    parser.add_argument("--ssh-command", dest='ssh_command', default='ssh', help='ssh command to use (default=ssh)')
     args = parser.parse_args()
 
     hosts = get_hosts()
-
     parsed_hosts = parse_hosts(hosts)
     
     selected = show_wofi(parsed_hosts)
-
-    selected_host = hosts[int(selected)]
-
-    ssh_to_host(selected_host, args.terminal)
\ No newline at end of file
+    
+    selected_host = selected.decode('utf-8').rstrip()
```

Comment:
This change breaks the script for me. Printing the resulting command in `ssh_to_host` gives:
`termite -e 'ssh 2'`

`show_wofi` returns the index of the selected host for me, not the host itself.

wofi v1.2.1 on Arch Linux